client/src/components/app-mount-notifier.test.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { I18nextProvider } from 'react-i18next';
+import { render, waitFor } from '@testing-library/react';
+import { Provider } from 'react-redux';
+
+import { i18nextCodes } from '../../../config/i18n/all-langs';
+import AppMountNotifier from './app-mount-notifier';
+import { createStore } from '../redux/createStore';
+import i18nTestConfig from '../../i18n/configForTests';
+
+jest.mock('react-ga');
+jest.unmock('react-i18next');
```

Comment:
`react-i18next` is mocked globally. It has to be unmocked here as the tests need use the actual implementation to dynamically change the language.